documentation/MediaInternet.json

```diff
@@ -0,0 +1,133 @@
+{
+  "type" : "Publication",
+  "identifier" : "c443030e-9d56-43d8-afd1-8c89105af555",
+  "status" : "DRAFT",
+  "owner" : "eier@example.org",
+  "publisher" : {
+    "type" : "Organization",
+    "id" : "http://example.org/org/123",
+    "labels" : {
+      "no" : "Eksempelforlaget"
+    }
+  },
+  "createdDate" : "2020-09-23T09:51:23.044996ZZ",
+  "modifiedDate" : "2020-09-23T09:51:23.044996ZZ",
+  "publishedDate" : "2020-09-23T09:51:23.044996ZZ",
+  "indexedDate" : "2020-09-23T09:51:23.044996ZZ",
+  "handle" : "http://example.org/handle/123",
+  "doi" : "http://example.org/doi/1231/98765",
+  "doiRequest" : {
+    "type" : "DoiRequest",
+    "status" : "REQUESTED",
+    "modifiedDate" : "2020-09-23T09:51:23.044996ZZ",
+    "createdDate" : "2020-09-23T09:51:23.044996ZZ",
+    "messages" : [ {
+      "type" : "DoiRequestMessage",
+      "text" : "Some Text",
+      "author" : "SomeAuthor",
+      "timestamp" : "2020-09-23T09:51:23.044996ZZ"
+    } ]
+  },
+  "link" : "http://example.org/link",
+  "entityDescription" : {
+    "type" : "EntityDescription",
+    "mainTitle" : "Hovedtittelen",
+    "alternativeTitles" : {
+      "en" : "English title"
+    },
+    "language" : "http://example.org/norsk",
+    "date" : {
+      "type" : "PublicationDate",
+      "year" : "2020",
+      "month" : "4",
+      "day" : "7"
+    },
+    "contributors" : [ {
+      "type" : "Contributor",
+      "identity" : {
+        "type" : "Identity",
+        "id" : "http://example.org/person/123",
+        "name" : "Navnesen, Navn",
+        "nameType" : "Personal",
+        "orcId" : "orc123",
+        "arpId" : "arp123"
+      },
+      "affiliations" : [ {
+        "type" : "Organization",
+        "id" : "http://example.org/org/123",
+        "labels" : {
+          "no" : "Eksempelforlaget"
+        }
+      } ],
+      "role" : "Creator",
+      "sequence" : 0,
+      "correspondingAuthor" : true
+    } ],
+    "npiSubjectHeading" : "010",
+    "tags" : [ "dokumenter", "publikasjoner" ],
+    "description" : "En streng som beskriver innholdet i dokumentet på en annen måte enn abstrakt",
+    "reference" : {
+      "type" : "Reference",
+      "publicationContext" : {
+        "type" : "Media",
+        "place" : "Elsewhere",
+        "programme" : "Norbert and Friends"
+      },
+      "doi" : "https://123123/213123.com",
+      "publicationInstance" : {
+        "type" : "MediaInternet",
```

Comment:
![bilde](https://user-images.githubusercontent.com/6313468/138121096-66ddd66e-7bb9-47e4-9527-a176c3aee03a.png)

I skissene ser det ut som at det er tenkt annerledes, hvor `publicationInstance.type` er en av: 
- Dokumenter
- Intervju
- Programledelse
- Programdeltagelse

Mens `Mediumtype` som virker å være det samme som `publicationInstance.contentType` for bl.a. artikkel i tidsskrift og kapittel